tools/as/as.cpp

```diff
@@ -79,6 +84,15 @@ int main(int argc, char** argv) {
             return 1;
           }
         } break;
+        case '-': {
+          // Long options
+          if (0 == strcmp(argv[argi], "--version")) {
+            printf("%s\r\n", kBuildVersion);
```

Comment:
I'm afraid I don't see the change yet.  Also make sure to remove all `\r`s added below.